src/main/java/duke/task/TaskConverter.java

```diff
@@ -0,0 +1,92 @@
+package duke.task;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Provides utilities for converting between task objects and their string representations.
+ * This class supports saving tasks to and loading tasks from a string-based format, facilitating
+ * persistence in text files or similar storage mechanisms.
+ */
+public class TaskConverter {

```

Comment:
Interesting class to make it clearer for converting task objects to strings which I think is a great idea!